syft/messaging/plan/procedure.py

```diff
@@ -74,74 +74,58 @@ def update_ids(
             from_worker: The previous worker that built the plan.
             to_worker: The new worker that is running the plan.
         """
-
-        # We operate on simplified content of Operation, hence all values should be simplified
-        from_workers_simplified = None
-        to_workers_simplified = None
-        if from_worker and to_worker:
-            from_workers_simplified = [sy.serde.msgpack.serde._simplify(None, from_worker)]
-            to_workers_simplified = [sy.serde.msgpack.serde._simplify(None, to_worker)]
-
-        from_ids_simplified = None
-        to_ids_simplified = None
-        if len(from_ids) and len(to_ids):
-            from_ids_simplified = [sy.serde.msgpack.serde._simplify(None, id) for id in from_ids]
-            to_ids_simplified = [sy.serde.msgpack.serde._simplify(None, id) for id in to_ids]
-
         for idx, operation in enumerate(self.operations):
-            if from_workers_simplified and to_workers_simplified:
-                operation = Procedure.replace_operation_ids(
-                    operation, from_workers_simplified, to_workers_simplified
-                )
-
-            if from_ids_simplified and to_ids_simplified:
-                operation = Procedure.replace_operation_ids(
-                    operation, from_ids_simplified, to_ids_simplified
-                )
+            # replace ids in the owner
+            owner = operation.cmd_owner
```

Comment:
What is the type of `operation` now? I can't find the definition here it must be in the unchanged files

Reply:
It's an `Operation` object.